test/index.js

```diff
@@ -457,4 +458,79 @@ module.exports = {
       });
     },
   },
+  multiVersionHelper: {
+    setUp(cb) {
+      this.req = {
+        get(key) {
+          return this[key];
+        },
+      };
+
+      cb();
+    },
+    default(test) {
+      test.expect(2);
+
+      const middleware = version.setBySemverAccept(this.supportedVersions, 'vnd.test');
+      this.req.accept = 'application/vnd.test.v1.0.0+json';
+      middleware(this.req, {}, () => {
+        test.equal(this.req.version, 'v1.0.0', 'Unexpected version');
+      });
+
+      const version1 = () => {
+        test.equal(this.req.version, 'v1.0.0');
+        test.done();
+      };
+
+      const version11 = () => {
+        // not called
```

Comment:
It might be worth adding a `test.ok(false)` in the body of this method to ensure that it's not called.

Reply:
cool! first time I use nodeunit, hehe